vignettes/function-flow.Rmd

```diff
@@ -0,0 +1,178 @@
+---
+title: "Function flow"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: vancouver.csl
+vignette: >
+  %\VignetteIndexEntry{Function flow}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Introduction
+
+This vignette describes the function conventions and function flow of
+the `osdc` package. The function convention sections go over how we name
```

Comment:
```suggestion
the osdc package. The function convention sections go over how we name
```

Just a small thing. Most package documentation in many packages refer to packages without backticks. It doesn't really matter which, but its good to be consistent.

Reply:
Alrighty. We'll do it without backticks 👍